Sources/FoundationEssentials/LockedState.swift

```diff
@@ -92,11 +92,13 @@ internal struct LockedState<State> {
         }
     }
 
-    // Ensures the managed state outlives the locked `body`.
+    // Ensures the managed state outlives the locked scope.
```

Comment:
Oops this was my mistake when I added this. You're right that this should be locked scope, not just body. Thanks y'all!